test_regress/t/t_dump_tree_dot.pl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(vlt => 1);
+
```

Comment:
This just tests dumping so you can just pick up an arbitrary existing source file, no need to add a new one unless there is some specific feature that I missed:
```suggestion

top_filename("t/t_bench_mux4k.v");
```

Reply:
Actually, let's pick a smaller example as even this is so big `xdot` struggles displaying it`